build.ps1

```diff
@@ -26,7 +26,7 @@ try {
     & docker run --rm `
         -v /var/run/docker.sock:/var/run/docker.sock `
         imagebuilder `
-        build --manifest "manifest.json" --path "$DockerfilePath" "$ImageBuilderCustomArgs"
+        build --manifest manifest.json --path "$DockerfilePath" ${ImageBuilderCustomArgs}.Split()
```

Comment:
I don't think calling split is a good idea.  The ImageBuilderCustomArgs may include quoted values itself.  Splitting in this scenario would not produce the desired behavior.  Why not use InvokeExpression instead?